minecraft_servers/crystopia/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+    "server_name": "crystopia",
+    "nice_name": "Crystopia",
+    "direct_ip": "crystopia.net",
+    "server_wildcards": [
+      "play.crystopia.net"
+    ],
+    "supported_languages": [
+       "en",
+       "de"
+    ],
+    "social": {
+      "web": "https://crystopia.net",
+      "web_shop": "https://shop.crystopia.net",
+      "web_support": "https://labymod.net/wiki",
+      "twitter": "crystopianet",
+      "discord": "https://dc.crystopia.net",
+      "youtube": "https://www.youtube.com/@crystopia"
+    },
+    "gamemodes": {
+      "smp": {
+        "name": "SMP", 
+        "color": "#FFA500",
+        "url": "https://www.crystopia.net/",
```

Comment:
Just out of interest: What is the specific context between this (general) website for this game mode?

Reply:
We don't have a separate page for game mode yet.

Reply:
Then remove?